cmd/gke-gcloud-auth-plugin/main.go

```diff
@@ -5,14 +5,24 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"os"
 	"os/exec"
 	"time"
 
 	"github.com/spf13/pflag"
+	"golang.org/x/oauth2"
 	"golang.org/x/oauth2/google"
 	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
 	clientauth "k8s.io/client-go/pkg/apis/clientauthentication/v1beta1"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 	"k8s.io/component-base/version/verflag"
+	"k8s.io/klog/v2"
+)
+
+const (
+	accessTokenEnvVar       = "GCLOUD_ACCESS_TOKEN"
```

Comment:
We can't pass secrets via environment variables.

Reply:
Moved to using a cache file..Updated in later commits.